layers/core_validation.cpp

```diff
@@ -15374,6 +15701,7 @@ bool CoreChecks::PreCallValidateCmdEndQueryIndexedEXT(VkCommandBuffer commandBuf
                              ") is greater or equal to the queryPool size (%" PRIu32 ").",
                              index, available_query_count);
         }
+#if defined(VK_EXT_transform_feedback)
```

Comment:
this is still inside the ifdef from line 15613

Reply:
Fixed in next patch